### PR TITLE
Feature/dapp run e2e test in environment nightly

### PR DESCRIPTION
**Pull request by weilbith**
Related to: #1584 

WIP - Opened here to trigger CI.

### CODE REVIEW

**Comment by weilbith** [comment]
I removed the trigger for the nightly workflow to make it testable here in the PR. [Here](https://app.circleci.com/pipelines/github/raiden-network/light-client/7117/workflows/9c38ca11-0241-4ee4-b836-2aace20a1a4c/jobs/44100) is a "proof" of a successful run. I'll re-establish the trigger now. 

PS: Please be aware that the end-to-end tests of the SDK are broken on `master` atm. So this PR is not fixing them and does fail for this job as well.

**Comment by weilbith** [comment]
Also note that we have now a new DockerHub repository. I'm about to open another PR to adopt the documentation and test script to make use of it per default.

File: e2e-environment/Dockerfile [hunk 1]
```diff
@@ -39,10 +39,10 @@ ARG CONTRACTS_VERSION
 ARG CONTRACTS_PACKAGE_VERSION
 ARG GETH_VERSION
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends supervisor python3-virtualenv \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends supervisor python3-virtualenv libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```
Comment (palango) [inline e2e-environment/Dockerfile:43]: Why do we need GTK in that image?

File: .circleci/config.yml [hunk 2]
```diff
@@ -27,11 +33,12 @@ executors:
   e2e-environment-executor:
     working_directory: ~/src
```
Comment (palango) [inline .circleci/config.yml:34]: Can we use `*working_dir` here as well?

File: e2e-environment/Dockerfile (hunk 1 above)
Comment (weilbith) [reply e2e-environment/Dockerfile:43]: This is for Cypress (did I miss this in the commit messages?).
Therefore I blindly followed [these instructions](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Linux) by them.

Comment (palango) [reply e2e-environment/Dockerfile:43]: Nah, I just didn't read them... 🙈

**Review by palango — APPROVED** [review]
Nice PR - especially the commit messages explain the problems and the solution very nice! 🏅

File: .circleci/config.yml (hunk 2 above)
Comment (weilbith) [reply .circleci/config.yml:34]: Yes. This configuration needs some love. The anchors are not used consequently etc. Do you want me to do it in this PR? :smiley:

Comment (palango) [reply .circleci/config.yml:34]: If you have more things in mind then a different PR, if you only want to fix that one, I'd just add it to this PR.

Comment (weilbith) [reply .circleci/config.yml:34]: Lets just do it. Always leave the code better than you found it.

Comment (weilbith) [reply .circleci/config.yml]: :heavy_check_mark:

**Comment by weilbith** [comment]
According to Paul his tip I'll rebase on latest master to fix the linting issue.

**Pull request merged**